src/components/Header/header.css

```diff
@@ -1,5 +1,5 @@
 .parallax-image {
-    background: url('../assets/images/library-img.jpg') no-repeat center center/ cover; 
+    background: url('../../assets/images/library-img.jpg') no-repeat center center/ cover; 
```

Comment:
You have used path alias in this project ig, will that work here? not sure, try it out and let me know :P

Reply:
Sure. Since it was in another branch, I did not use it here.